Kickstarter-iOS/Views/Controllers/DeprecatedRewardPledgeViewControllerTests.swift

```diff
@@ -0,0 +1,352 @@
+import Library
```

Comment:
This DIFF is showing renaming `RewardPledgeViewControllerTests.swift` to `DeprecatedRewardPledgeViewControllerTests.swift`. Unfortunately it shows the diff because we also added new implementation of `RewardPledgeViewController.swift` 🤦‍♂️